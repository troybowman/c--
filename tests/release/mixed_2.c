../decl/mixed_2.c