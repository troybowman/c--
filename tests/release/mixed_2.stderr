Unterminated comment, line 53
error: variable array redeclared at line 4 (previous declaration at line 3)
error: variable func8 redeclared at line 18 (previous declaration at line 17)
error: variable func9 redeclared at line 20 (previous declaration at line 19)
error: function func2 redeclared at line 24 (previous declaration at line 9)
error: function func3 redeclared at line 24 (previous declaration at line 10)
error: function func4 redeclared at line 24 (previous declaration at line 11)
error: function func5 redeclared at line 24 (previous declaration at line 12)
error: function func6 redeclared at line 24 (previous declaration at line 13)
error: variable func7 redeclared at line 26 (previous declaration at line 15)
syntax error, line: 30
syntax error, line: 32
syntax error, line: 34
